tests/TestingAppHost1/TestingAppHost1.AppHost/appsettings.Development.json

```diff
@@ -4,5 +4,9 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "Parameters": {
+    "pgsqluser": "test",
+    "pgsqlpass": "pass123"
```

Comment:
Do we need these? I might trigger a cred scan warning. It might be better to inject the parameter dynamically at runtime (with a random value).